src/test/resources/__files/api/v1/project/get-all-projects-with-pagination.json

```diff
@@ -0,0 +1,123 @@
+[
```

Comment:
Is this just a copy of the `get-all-projects.json` file?  If so what is the benefit of duplicating it?

Reply:
It's got one less project so if some calls the api just "/api/v1/project", the count of projects returned will be different and the test will fail.